docs/dev-tools/zombienet/testing.md

```diff
@@ -0,0 +1,162 @@
+# Testing DSL
+
+Zombienet provides a Domain Specific Language (DSL) for writing tests. The DSL is designed to be human-readable and allows you to write tests using natural language expressions. Using this DSL, you can define assertions and tests against the spawned network. This way, users can evaluate different metrics, such as:
+
+- On chain storage
```

Comment:
Maybe in this list we should provide a bit more description of each component?  Like

- **On-chain storage** - the storage of each of the chains running via Zombienet
...